.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+### Changes
+
+Please describe both what is changing and why this is important. Include:
+
+- Endpoints added, deleted, deprecated, or changed
+- Classes and methods added, deleted, deprecated, or changed
+- Screenshots of new or changed UI, if applicable
+- A summary of usage if this is a new feature or change to a public API (this should also be added to relevant documentation once released)
+
+### References
+
+Please include relevant links supporting this change such as a:
+
+- support ticket
+- community post
+- StackOverflow post
+- support forum thread
+
+Please note any links that are not publicly accessible.
+
+### Testing
+
+Please describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
+
+* [ ] This change adds unit test coverage
+* [ ] This change has been tested on the latest version of the platform/language or why not
+
+### Checklist
+
+* [ ] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
+* [ ] All existing and new tests complete without errors
+* [ ] All active GitHub checks have passed
```

Comment:
If we are doing to do the checklist approach, were missing a few. Like have the docs been updated? Are the commits following our standards? etc.

Reply:
Personally, I am not a big fan of the checklist approach. People just check them but don't generally provide the context.

Reply:
I think it might be useful especially for new or occasional contributors so they are aware of the requirements + they can ensure that their PR matches our expectations and provides a required level of details.

If not a checklist then maybe just a list that would be a **comment** inside the template so it's not published but it's always there when you want to edit a description.

Reply:
If it's a checklist, it should be someone broad and only the important items. I like the last two listed here. Docs could be added